xml/System.Data.SqlClient/SqlCommand.xml

```diff
@@ -1637,7 +1701,7 @@ SELECT * FROM dbo.Customers WHERE CustomerID = @CustomerID
 >   
 >  For example, with a 30 second time out, if <xref:System.Data.SqlClient.SqlDataReader.Read%2A> requires two network packets, then it has 30 seconds to read both network packets. If you call <xref:System.Data.SqlClient.SqlDataReader.Read%2A> again, it will have another 30 seconds to read any data that it requires.  
   
-```  
+```csharp
 using System;  
 using System.Data.SqlClient;  
 ///   
```

Comment:
There should also be an `-or` between lines 2218 and 2220 to differentiate the exception conditions.